components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -29,6 +27,12 @@ import appConfigPath from './user_config/app-config.yaml';
 // selects auto domain then we should automatically supply the suffix
 // <hostname>.endpoints.<Project>.cloud.goog
 
+const IngressType  = {
+  BasicAuth: 'Login with Username Password',
+  Iap: 'Login with GCP Iap',
+  Skip: 'Skip Endpoint'
```

Comment:
What is skip doing?

In the case of Skip endpoint can we still setup IAP but with fake OAuth client credentials?

Should we call it something like

'Iap : partial setup'
or

'Iap:  Finish OAuth setup later'

I know its not great but I'd prefer a name that is accurate and consistent with long term plans.

Reply:
We can change this to something like shared OAuth client later when that's available.